test/tests/userobjects/moab_skinner/expanding-cube/all_bins.i

```diff
@@ -0,0 +1,139 @@
+[Mesh]
+ [cube]
+    type = FileMeshGenerator
+    file = '../../../neutronics/meshes/tet_cube.e'
+ []
+ [id1]
+    type = ParsedSubdomainMeshGenerator
+    input = cube
+    combinatorial_geometry = 'z < 0.0'
+    block_id = 1
+ []
+ [id2]
+    type = ParsedSubdomainMeshGenerator
+    input = id1
+    combinatorial_geometry = 'z >= 0.0'
+    block_id = 3
+ []
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+  use_displaced_mesh = true
+[]
+
+[Problem]
+  type = FEProblem
+  solve = false
+[]
+
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
+[]
+
+[AuxVariables]
+  [rho]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [temp]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [rho_bins]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [temp_bins]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [subdomain_bins]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [bins]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+[]
+
+[AuxKernels]
+  [rho]
+    type = FunctionAux
+    variable = rho
+    function = rho
+    execute_on = timestep_begin
+  []
+  [temp]
+    type = FunctionAux
+    variable = temp
+    function = temp
+    execute_on = timestep_begin
+  []
+  [rho_bins]
+    type = SkinnedBins
+    variable = rho_bins
+    skinner = moab
+    skin_by = density
+  []
+  [temp_bins]
+    type = SkinnedBins
+    variable = temp_bins
+    skinner = moab
+    skin_by = temperature
+  []
+  [subdomain_bins]
+    type = SkinnedBins
+    variable = subdomain_bins
+    skinner = moab
+    skin_by = subdomain
+  []
+  [bins]
+    type = SkinnedBins
+    variable = bins
+    skinner = moab
+  []
+[]
+
+[Functions]
+  [temp]
+    type = ParsedFunction
+    expression = 400+x*100+100*t
```

Comment:
It looks like as the mesh is deforming, that the deformation doesn't cause any change in the actual skinning (not that anything is wrong with your code, but I think we can design this test better). Can you modify the functions used for temperature and density in order to make something that will change how the bins are assigned based on the mesh movement? I'd remove the time dependence, and make the temperature dependence such that, as the mesh moves, the skins also change.

Reply:
You may just want to use the `temperature_min` and `temperature_max` to exactly enclose the min/max temperature so that you can see the bins move as the mesh moves.